src/store/PeopleContext.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+export const PeopleContext = React.createContext({
+  people: [] as Person[],
+  loading: false,
+  errorMessage: '',
+  loadPeople: async () => {},
```

Comment:
The default implementation of `loadPeople` in the context is an empty async function. Consider providing a more meaningful default or ensuring that this function is always overridden by the provider.